### PR TITLE
FISH-8879 upgrade jakarta authorization api to 3

### PR DESCRIPTION
**Pull request by luiseufrasio**
## Description
For Payara 7:
Upgrade to the final version if available, otherwise the “best” milestone release.

### Blockers
None

### New tests
None

### Testing Performed
build and run

### Testing Environment
Zulu JDK 11.0.11 on Windows 11 with Maven 3.8.6

### CODE REVIEW

**Review by Pandrex247 — DISMISSED** [review]

**Comment by luiseufrasio** [comment]
jenkins test please

File: core/pom.xml [hunk 1]
```diff
@@ -100,7 +100,7 @@
         <jakarta.enterprise.deploy-api.version>1.7.2</jakarta.enterprise.deploy-api.version>
         <jakarta.resource-api.version>2.1.0</jakarta.resource-api.version>
         <jakarta.annotation-api.version>3.0.0</jakarta.annotation-api.version>
-        <exousia.version>2.1.2</exousia.version>
+        <exousia.version>3.0.0-M3</exousia.version>
```
Comment (Pandrex247) [inline core/pom.xml:103]: I'd be careful about blleding PRs together - this upgrade breaks quicklook.
Might be worth doing Exousia in a separate PR

**Review by Pandrex247 — APPROVED** [review]
